src/pages/index.astro

```diff
@@ -12,6 +43,17 @@ import MainLayout from '../layouts/Main.astro'
     <title>PatternFly</title>
   </head>
   <body>
-    <MainLayout title="PatternFly X Astro"> Page content </MainLayout>
+    <MainLayout title="PatternFly X Astro">
+      <PropsTable
+        componentName="Alert"
+        componentDescription="The main alert component."
+        componentProps={propsData}
+      />
+      <PropsTable
+        componentName="AlertGroup"
+        componentDescription="The container to render a list of alerts."
+      />
+      <PropsTable componentName="AlertTitle" />
+    </MainLayout>
```

Comment:
Did you intend to keep these changes in the PR?

Reply:
Just for now, to show what the various renders look like in case anyone had thoughts/concerns or anything (also to get design input on the Label choices, though we can do that another time if need be; I can otherwise back that file change out in my next push if things look good otherwise to you)